src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -354,6 +354,18 @@ public DocumentConventions()
         private bool _disableTcpCompression;
         private bool _disposeCertificate;
 
+        internal bool _includePromotablesInTopology;
+
+        internal bool IncludePromotablesInTopology
```

Comment:
would like to avoid using DocumentConventions as DTO

Reply:
removed